examples/substrate-react-example/src/substrate-lib/SubstrateContext.tsx

```diff
@@ -12,16 +10,37 @@ import {
   deposit
 } from "@buildwithsygma/sygma-sdk-core";
 
-import config, { SubstrateConfigAssetType } from "../config";
+import config from "../config";
 
 const connectedSocket = config.provider_socket;
+
+type State = {
+  socket: any;
+  jsonrpc: any | null;
+  keyring: any | null;
+  keyringState: any | null;
```

Comment:
Please do not use `any`
btw `null` is in `any` too - [example](https://www.typescriptlang.org/play?#code/DYUwLgBAbghgTgSxgI1ALgjAdgTwNwBQBsiKoEAvBFgK7DCElKoiUQBEAEiPQPYQB1XnGAATdo3jNyVAIyTSLNu2QTiUsiADUWvEA)